Set Facebook app id from code

rohan-dhamal-clarice opened the pull request:
I could not reproduce this issue. However, it seems that the crash occurred because applicationId was not found. Though meta-data already has applicationId configured, setting it from within code might fix this issue. Per SDK, When applicationId is already set from code, then Facebook SDK won't read applicationId from meta-data in Manifest file.

JIRA: https://openedx.atlassian.net/browse/MOB-1348

@nasthagiri Please review.
cc @shahidtamboli @aleffert

aleffert commented:
Looks good. Also, if it's guaranteed to read this is it safe to get rid of the mention in the android manifest? That would make the build script a little simpler.

rohan-dhamal-clarice commented:
@aleffert I have updated build script and also removed Facebook app Id from <code>AndroidManifest</code>. Tested these changes and all works well.
Please review.

aleffert commented:
:+1:
